src/status_im2/contexts/chat/home/chat_list_item/style.cljs

```diff
@@ -1,7 +1,7 @@
 (ns status-im2.contexts.chat.home.chat-list-item.style
   (:require [quo2.foundations.colors :as colors]))
 
-(def container
+(defn container []
```

Comment:
I personally prefer to always use defn for styling definitions since we may need to pass some args to it at any time later. I know we can then change it, but to be consistent. Just my opinion, what do you think?

Reply:
in 99% of cases, it will never change, and also by using def you can already see from view if the style is dynamic or not, so you don't need to go inside to see that, so better to use def if it's not dynamic